scripts/start.js

```diff
@@ -42,11 +60,22 @@ switch(argv.mode) {
     concurrently(
       [
         startSources,
-        startCore,
+        startCoreProd,
+        startReact,
```

Comment:
is `startCore` dan nog wel ergens nodig?

Reply:
Ja, bij `yarn start:all`